website/content/api-docs/secret/aws.mdx

```diff
@@ -585,3 +585,136 @@ $ curl \
   }
 }
 ```
+
+## Create Static Role
+This endpoint creates or updates static role definitions. A static role is a 1-to-1 mapping
+with an AWS IAM User, which will be adopted and managed by Vault, including rotating it according
+to the configured `rotation_period`.
+
+<Note>
+
+Vault will create a new credential upon configuration, and if the maximum number of access keys already exist, Vault will rotate the oldest one. Vault must do this to know the credential. 
+
+At each rotation, Vault will rotate the oldest existing credential.
+
+</Note>
+
+| Method | Path                      |
+| :----- | :------------------------ |
+| `POST` | `/aws/static-roles/:name` |
+
+### Parameters
+
+- `name` `(string: <required>)` – Specifies the name of the role to create. This
+is specified as part of the URL.
+
+- `username` `(string: <required>)` – Specifies the username of the IAM user.
+
+- `rotation_period` `(string/int: <required>)` – Specifies the amount of time
+Vault should wait before rotating the password. The minimum is 1 minute. Can be
+speciied in either `24h` or `86400` format.
```

Comment:
Might help users to link to the [duration-format](https://developer.hashicorp.com/vault/docs/concepts/duration-format) similar to some other APIs that have duration fields.

Reply:
done!